lang/lang_main.php

```diff
@@ -9,29 +9,44 @@
  * @version 3.3.0
  */
 
+function load_lang_file($ui_lang, $filename, $parent_dir = ".") {
+    global $lang;
```

Comment:
Ok, there's a radical way of avoiding this complain, by adding:

```
global $lang;
```
At the start of ALL php files for all languages (`lang/*/lang_*.php`)

@darknoon29 this means that translators need to keep that in mind, of course.

If you want this approach, please ping me and I will start updating all files.

Reply:
Needless to say, it would be even better changing the translate system so we don't have the translations as PHP files.

We could open an issue for that and handle it on a separate PR, as it's a big refactor.

Reply:
Hi, i use transifex to manage translations : https://www.transifex.com/ogsteam/ogspy/dashboard/
by using their tool i'm able to generate all language files in one command ;-)

Reply:
Ah, good, I had a question at the forum about this, precisely (as I saw the English and Spanish translations are not complete for the 3.3.5 version.

So, if the PHP files are generated by transiflex, forcing `global` there is not an option. The only way I see to fix this is to have the variable declared outside the function and initialized to an empty string.

Let's see if that way the bot does not complain.

Reply:
Don't worry about this global, we use it like this at least 30 times into OGSpy. We've got a ambitious refactoring (Check our roadmaps) on going and this can be managed in future versions :-)

Reply:
And yet again, no, the `global` declaration must be either inside the function or the included files.

Let's initialize the array if it is empty. Maybe that will do the trick.